templates/components/amp/common/share.html

```diff
@@ -0,0 +1,19 @@
+{{#if settings.add_this.buttons}}
+    <ul class="socialLinks" data-addthis:url="{{url}}" data-addthis:title="{{title}}">
+        {{#each settings.add_this.buttons}}
+          {{#if service '!==' 'google'}}
+            {{#if service '!==' 'print'}}
+              {{#if service '!==' 'facebook'}}
```

Comment:
Actually, maybe it would be better to put this logic into a helper and just return the computed html to the template. Easier to read, and likely less typing. Would that work?